roms/vgabios/config.mak

```diff
@@ -1,5 +1,5 @@
 # Automatically generated by configure - do not modify
-SRC_PATH=/opt/qemu-1/roms/vgabios
+SRC_PATH=/opt/s2earm/source/s2e/qemu/roms/vgabios
```

Comment:
Please squash all commits, then clean up files that should not be modified, then redo new commits, logically separated.